mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/SendScreenV2.tsx

```diff
@@ -0,0 +1,577 @@
+  const client = useWhaleApiClient()
+  const { isLight } = useThemeContext()
+  const { getTokenPrice } = useTokenPrice()
+  const { fetchWalletAddresses } = useWalletAddress()
+  const { getDisplayUtxoWarningStatus } = useDisplayUtxoWarning()
+  const toast = useToast()
+  const TOAST_DURATION = 2000
+
+  const tokens = useSelector((state: RootState) => tokensSelector(state.wallet))
+  const addressBook = useSelector((state: RootState) => state.userPreferences.addressBook)
+  const walletAddress = useSelector((state: RootState) => state.userPreferences.addresses)
+  const hasPendingJob = useSelector((state: RootState) => hasTxQueued(state.transactionQueue))
+  const hasPendingBroadcastJob = useSelector((state: RootState) => hasBroadcastQueued(state.ocean))
+  const DFIUtxo = useSelector((state: RootState) => DFIUtxoSelector(state.wallet))
+  const DFIToken = useSelector((state: RootState) => DFITokenSelector(state.wallet))
+
+  const [token, setToken] = useState(route.params?.token)
+  const [matchedAddress, setMatchedAddress] = useState<LocalAddress>()
+  const [fee, setFee] = useState<BigNumber>(new BigNumber(0.0001))
+  const [jellyfishWalletAddress, setJellyfishWalletAddresses] = useState<string[]>([])
+  const [transactionCardStatus, setTransactionCardStatus] = useState(TransactionCardStatus.Default)
+
+  // form
+  const { control, setValue, formState, getValues, trigger, watch } = useForm({ mode: 'onChange' })
+  const { address } = watch()
+  const amountToSend = getValues('amount')
+
+  const {
+    isConversionRequired,
+    conversionAmount
+  } = useConversion({
+    inputToken: {
+      type: token?.id === '0_unified' ? 'utxo' : 'others',
+      amount: new BigNumber(amountToSend)
+    },
+    deps: [amountToSend, JSON.stringify(token)]
+  })
+
+  const debounceMatchAddress = debounce(() => {
+    if (address !== undefined && addressBook !== undefined && addressBook[address] !== undefined) {
+      setMatchedAddress(addressBook[address])
+    } else if (address !== undefined && walletAddress !== undefined && walletAddress[address] !== undefined) {
+      setMatchedAddress(walletAddress[address])
+    } else if (address !== undefined && jellyfishWalletAddress.includes(address)) {
+      // wallet address that does not have a label
+      setMatchedAddress({
+        address,
+        label: 'Saved address',
+        isMine: true
+      })
+    } else {
+      setMatchedAddress(undefined)
+    }
+  }, 200)
+
+  const reservedDFI = 0.1
+  const isReservedUtxoUsed = getDisplayUtxoWarningStatus(new BigNumber(amountToSend))
+  const amountInUSDValue = useMemo(() => {
+    if (token === undefined || isNaN(amountToSend) || (amountToSend === '')) {
+      return new BigNumber(0)
+    }
+
+    return getTokenPrice(token.symbol, amountToSend, token.isLPS)
+  }, [amountToSend, token])
+
+  const { infoText, infoTextThemedProps } = useMemo(() => {
+    let infoText; let themedProps; let status = TransactionCardStatus.Default
```

Comment:
let's not do this style haha

Reply:
All declarations on one line

Reply:
thought you're open minded haha, updated!